src/django/api/constants.py

```diff
@@ -81,6 +81,7 @@ class FeatureGroups:
 
 class FacilityListStatus:
```

Comment:
This class isn't referred to be anything except the `AdminFacilityListView` `get_queryset` method. Perhaps it can be removed in favor of `FacilityList` statuses.

Reply:
[07b1549](https://github.com/open-apparel-registry/open-apparel-registry/pull/2362/commits/07b15492fa12345e790333ee19f372fc8e798b96)